src/pages/DepositCryptoPage/index.tsx

```diff
@@ -0,0 +1,286 @@
+import {Dialog} from '@lykkex/react-components';
+import {observable} from 'mobx';
+import {inject, observer} from 'mobx-react';
+import * as React from 'react';
+import * as CopyToClipboard from 'react-copy-to-clipboard';
+import {RouteComponentProps} from 'react-router-dom';
+import {RootStoreProps} from '../../App';
+import {APPSTORE_LINK, GOOGLEPLAY_LINK} from '../../components/Apps';
+import ClientDialog from '../../components/ClientDialog';
+import Spinner from '../../components/Spinner';
+import WalletTabs from '../../components/WalletTabs/index';
+import {ROUTE_WALLETS_TRADING} from '../../constants/routes';
+import {STORE_ROOT} from '../../constants/stores';
+import {DialogModel} from '../../models';
+import {DialogConditionType} from '../../models/dialogModel';
+
+// tslint:disable-next-line:no-var-requires
+const QRCode = require('qrcode.react');
+
+import './style.css';
+
+interface DepositCryptoPageProps
+  extends RootStoreProps,
+    RouteComponentProps<any> {}
+
+export class DepositCryptoPage extends React.Component<DepositCryptoPageProps> {
+  @observable copiedToClipboardText = '';
+  @observable addressLoaded = false;
+
+  readonly assetStore = this.props.rootStore!.assetStore;
+  readonly dialogStore = this.props.rootStore!.dialogStore;
+  readonly uiStore = this.props.rootStore!.uiStore;
+
+  componentDidMount() {
+    const {assetId} = this.props.match.params;
+    const asset = this.assetStore.getById(assetId);
+
+    const clientDialog = this.dialogStore.pendingDialogs.find(
+      (dialog: DialogModel) =>
+        dialog.conditionType === DialogConditionType.Predeposit
+    );
+    if (clientDialog) {
+      clientDialog.visible = true;
+    }
+    if (assetId === 'ETH' || (asset && asset.name === 'ETH')) {
```

Comment:
I think it make sense to add helper function in assetStore isEth because you already have same in other tsx file